@guitargeek there was also a problem in another file, I forgot to note it before:
```
src/RecoEgamma/EgammaTools/interface/Utils.h: In function 'void writeValueMap(edm::Event&, const edm::Handle<HandleType>&, const std::vector<ValueType>&, const string&)': src/RecoEgamma/EgammaTools/interface/Utils.h:15:41: error: 'ValueMap' is not a member of 'edm' auto valMap = std::make_unique<edm::ValueMap<ValueType>>(); ^~~~~~~~ src/RecoEgamma/EgammaTools/interface/Utils.h:15:24: error: parse error in template argument list auto valMap = std::make_unique<edm::ValueMap<ValueType>>(); ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ src/RecoEgamma/EgammaTools/interface/Utils.h:15:62: error: expected primary-expression before ')' token auto valMap = std::make_unique<edm::ValueMap<ValueType>>(); ^ src/RecoEgamma/EgammaTools/interface/Utils.h:16:19: error: 'ValueMap' in namespace 'edm' does not name a template type typename edm::ValueMap<ValueType>::Filler filler(*valMap); ^~~~~~~~ src/RecoEgamma/EgammaTools/interface/Utils.h:16:27: error: expected unqualified-id before '<' token typename edm::ValueMap<ValueType>::Filler filler(*valMap); ^ src/RecoEgamma/EgammaTools/interface/Utils.h:17:5: error: 'filler' was not declared in this scope filler.insert(handle, values.begin(), values.end()); ^~~~~~ src/RecoEgamma/EgammaTools/interface/Utils.h:17:5: note: suggested alternative: 'fileno' filler.insert(handle, values.begin(), values.end()); ^~~~~~ fileno gmake: *** [config/SCRAM/GMake/Makefile.chk_headers:44: tmp/slc6_amd64_gcc700/check_header/RecoEgamma/EgammaTools/interface/Utils.h] Error 1 
```

--- a/RecoEgamma/EgammaTools/interface/Utils.h
+++ b/RecoEgamma/EgammaTools/interface/Utils.h
@@ -2,9 +2,12 @@
 #define RecoEgamma_EgammaTools_Utils_H
 
 #include "DataFormats/Common/interface/Handle.h"
+#include "DataFormats/Common/interface/ValueMap.h"
 #include "FWCore/Framework/interface/Event.h"
 
 #include <memory>
+#include <string>
+#include <vector>
 
 template<typename ValueType, class HandleType>
 void writeValueMap(edm::Event &iEvent,

